examples/example_onemod_pipeline_representation.json

```diff
@@ -0,0 +1,153 @@
+{
+    "name": "pipeline_name",
+    "directory": "path/to/directory",
+    "ids": [
+        "age_group_id",
+        "location_id",
+        "sex_id",
+        "year_id"
+    ],
+    "id_subsets": {
+        "age_group_id": [
+            8,
+            9,
+            10
+        ],
+        "location_id": [
+            13,
+            14
+        ],
+        "sex_id": [
+            1,
+            2
+        ],
+        "year_id": [
+            1980,
+            1981,
+            1982
+        ]
+    },
+    "observations_column": "obs",
+    "predictions_column": "pred",
+    "holdout_columns": [
+        "holdout1",
+        "holdout2"
+    ],
+    "test_column": "test",
+    "mtype": "binomial",
```

Comment:
We could change this to `model_type` to be more descriptive like the other items

Reply:
Sounds good